_posts/2020-04-08-tf-federated-intro/tf-federated-intro.Rmd

```diff
@@ -0,0 +1,464 @@
+---
+title: "A first look at federated learning with TensorFlow"
+description: > 
+  The term "federated learning" was coined to describe a form of distributed model training where the data remains on the clients, i.e., is never shipped to the coordinating server. In this post, we introduce central concepts and run first experiments with TensorFlow Federated, using R.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatffederatedintro
+date: 04-08-2020
+categories:
+  - Privacy & Security
+  - TensorFlow/Keras
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/federated_learning.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Here, stereotypically, is the process of applied deep learning: Gather/get data;
+iteratively train and evaluate; deploy. Repeat (or have it all automated as a
+continuous workflow anyway). We often discuss training and evaluation;
+deployment matters to various degrees, depending on the circumstances. But the
+data often is just assumed to be there: All together, in one place. (On your
+laptop; on a central server; in some cluster in the cloud.) In real life though,
+data could be all over the world: on smartphones for example, or on IoT devices.
+There's a lot of reasons why we don't want to ship all that data to some central
```

Comment:
Interesting! I think I may be doing the "hypercorrect" thing sometimes, from a non-native speaker's point of view ... (like, I normally use "data" as singular, probably because at some point in the past I must have found that very strange)